proposals/wac-ucr/index.bs

```diff
@@ -869,6 +1204,15 @@ payment for small enough sums. Users of the Solid Music App would thus have
 one more song to choose from amongst the large number made available by
 musicians worldwide.
 
+<div class="assertion">
+
+* [[#req-read]]
+* [[#req-conditional-action]]
+* [[#req-conditional-relationship]]
+* [[#req-vc]]
+
```

Comment:
The requirements here are missing on the obvious one
"The system shall allow the client to read the access control rules in order for it to be able to make a decision on what credentials (if any) to use."

Reply:
Resolved in https://github.com/solid/authorization-panel/commit/f3a636936623033bad279d3cea53dc35c2d398cc, per [discussion](https://github.com/solid/authorization-panel/pull/167/files?short_path=e66d5b0#diff-e66d5b08e777fcb77d751f43b35a40f779e28776d19fde98601f334c1105e56a)